dimod/binary/binary_quadratic_model.py

```diff
@@ -1823,13 +1823,37 @@ class Float64BQM(BQM, default_dtype=np.float64):
     pass
 
 
-def Binary(label: Variable, bias: Bias = 1,
+@unique_variable_labels
+def Binary(label: Optional[Variable] = None, bias: Bias = 1,
            dtype: Optional[DTypeLike] = None) -> BinaryQuadraticModel:
+    """Return a :class:`.BinaryQuadraticModel` instance representing a binary variable.
+
+    Args:
+        label: Optional hashable label to identify the variable.
```

Comment:
```
label (optional): ...
```

Reply:
Is this necessary in docstring if the type hint is present?

Reply:
Nope, nor is it required in the text :shrug:

Reply:
Fair point!